### PR TITLE
Fixed grammar issues in the fixtures docs.

### PR DESCRIPTION
**Pull request by boxed**
#### Trac ticket number
N/A

#### Branch description
Fixed grammar issues in the fixtures docs.

#### Checklist
- [x] This PR targets the `main` branch. 
- [x] The commit message is written in past tense, and ends with a period.

### CODE REVIEW

**Review by shangxiao — COMMENTED** [review]
Thanks 👍 

We would need to be consistent with statements for all headings. I've never read that page tbh and it looks weird that they're all questions 😅

**Comment by boxed** [comment]
I don't think being consistent is the best approach. We want good titles, and that can be different depending on the subject matter. The Django docs overall has both questions and statements, which is something I checked before writing this PR.

**Comment by boxed** [comment]
I changed one that was a question that didn't make much sense. The other ("What is a fixture?") I think makes a lot of sense as a question I think, and becomes worse as "What fixtures are". I didn't sleep much last night so maybe my English skills are failing me and there's a better non-question name though :P If so, please help me and I'll update.

File: docs/topics/db/fixtures.txt [hunk 1]
```diff
@@ -40,8 +40,8 @@ or to provide some :ref:`initial data <initial-data-via-fixtures>` using the
 
     django-admin loaddata <fixture label>
 
-Where Django looks for fixtures?
-================================
+Where does Django look for fixtures?
```
Comment (nessita) [inline docs/topics/db/fixtures.txt]: I would propose we remove all question marks, so for this remaining case, how about:
```suggestion
How fixtures are discovered
```

**Review by nessita — APPROVED** [review]
Thank you @boxed, I agree this is a nice improvement. I pushed a couple of extra edits to get rid of the question marks completely.

**Comment by boxed** [comment]
Thanks!

**Pull request merged**